### DISCUSSION TITLE
pass coordinates to ufunc

### DISCUSSION THREAD

**Discussion by espiritocz** [pydata/xarray#5602] [General]
hi,
i have a=xr.DataArray with lon,lat,time coordinates.
the only thing i want to do is apply function per each lon,lat.
i know this should be rather simple, but got stuck in it..
i.e.
def func(lat = 15.1, lon = 30.3, time = [Timestamp('2014-11-05 11:26:38'), Timestamp('2014-11-29 11:26:38']):
    return [1, 2]

and i want to:
xr.apply_ufunc(func, a.lat, a.lon)

thank you in advance..

**Comment by max-sixty** [comment]
Please check out the "discussion norms", including providing a reproducible example
